### PR TITLE
docs: add code snippets for imported tensorflow model

### PR DESCRIPTION
**Pull request by ashleyxuu**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-bigquery-dataframes/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes internal #338878382 🦕

### CODE REVIEW

File: samples/snippets/imported_tensorflow_model_test.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    PROJECT_ID = os.getenv("GOOGLE_CLOUD_PROJECT", "bigframes-dev")
+    LOCATION = "US"
+    MODEL_PATH = "gs://cloud-training-demos/txtclass/export/exporter/1549825580/*"
+
+    # [START bigquery_dataframes_imported_tensorflow_tutorial_import_tensorflow_models]
+    import bigframes
+    from bigframes.ml.imported import TensorFlowModel
+
+    bigframes.options.bigquery.project = PROJECT_ID
+    bigframes.options.bigquery.location = LOCATION
```
Comment (tswast) [inline samples/snippets/imported_tensorflow_model_test.py:30]: No need for this to be a constant. Our goal is to be as copy-pasteable as possible and this adds a barrier to that.

Just say `"US"` and add a comment saying they can change it to one of the valid locations (link to BQ locations).

File: samples/snippets/imported_tensorflow_model_test.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+    imported_tensorflow_model = TensorFlowModel(model_path=MODEL_PATH)
```
Comment (tswast) [inline samples/snippets/imported_tensorflow_model_test.py:32]: Please add a comment showing an example string. I want them to see that they need the important information, like `gs://`, the bucket name, and the `*` wildcard. `model_path=MODEL_PATH,  # e.g. "gs://bucket-name/path/to/model/*"``

File: samples/snippets/imported_tensorflow_model_test.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+
+def test_imported_tensorflow_model() -> None:
+    # Determine project id, in this case prefer the one set in the environment
+    # variable GOOGLE_CLOUD_PROJECT (if any)
+    import os
+
+    PROJECT_ID = os.getenv("GOOGLE_CLOUD_PROJECT", "bigframes-dev")
+    LOCATION = "US"
+    MODEL_PATH = "gs://cloud-training-demos/txtclass/export/exporter/1549825580/*"
```
Comment (tswast) [inline samples/snippets/imported_tensorflow_model_test.py:23]: Is this a public dataset? If so, let's just include it in the code sample itself.

**Review by tswast — CHANGES_REQUESTED** [review]

File: samples/snippets/imported_tensorflow_model_test.py (hunk 3 above)
Comment (ashleyxuu) [reply samples/snippets/imported_tensorflow_model_test.py:23]: I don't think it is a public dataset, I put `"gs://bucket/path/to/saved_model/*"` in the comment

**Review by tswast — APPROVED** [review]

**Pull request merged**